Configure

```diff
@@ -617,6 +617,9 @@ my @disable_cascades = (
 
     "cmp"               => [ "crmf" ],
 
+    # no-fips causes fips-securitychecks to also be disabled.
+    "fips"              => [ "fips-securitychecks" ],
```

Comment:
Looks fine except I would drop the comment. Exactly the same comment could be made against every entry in `disable_cascades` so there seems little point in singling this one out in particular.

Reply:
Comment dropped